config/src/config/mod.rs

```diff
@@ -398,22 +398,12 @@ impl NodeConfig {
             )?;
         }
 
-        let mut network_ids = HashSet::new();
         if let Some(network) = &mut self.validator_network {
             network.load_validator_network()?;
             network.mutual_authentication = true; // This should always be the default for validators
-            network_ids.insert(network.network_id);
         }
         for network in &mut self.full_node_networks {
             network.load_fullnode_network()?;
-
-            // Check a validator network is not included in a list of full-node networks
-            let network_id = network.network_id;
-            invariant(
```

Comment:
I assume you removed this because the call to `network.load_fullnode_network()?;` above will panic if the network is a validator network?

If that's the case, would you mind adding a new unit test to the bottom of this file that calls `validate_network_configs` with an invalid fullnode network config that will trigger the error? Just want to make sure the check is not lost in the future 😄

Reply:
Thanks for your review. I've added a simple test for this, please take a look.